faker/providers/person/pl_PL/__init__.py

```diff
@@ -774,3 +774,26 @@ def pwz_nurse(self, kind='nurse'):
         kind_char = 'A' if kind == 'midwife' else 'P'
 
         return '{:02d}{}{}'.format(region, ''.join(map(str, core)), kind_char)
+
+    def nip(self):
+        """
+        Returns 10 digit of Number of tax identification.
+        Polish: Numer identyfikacji podatkowej (NIP).
+
+        https://pl.wikipedia.org/wiki/NIP
+
+        """
+        nip = []
+        for _ in range(9):
+            nip.append(self.random_digit())
```

Comment:
The last three lines can be expressed more succinctly with a very slight performance improvement. Use a list comprehension like so:

```
nip = [self.random_digit() for _ in range(9)]
```

Reply:
I agree with your statement, change done